src/binary-reader-objdump.cc

```diff
@@ -1029,10 +1029,20 @@ Result BinaryReaderObjdump::OnType(Index index,
     printf("%s", GetTypeName(param_types[i]));
   }
   printf(") -> ");
-  if (result_count) {
-    printf("%s", GetTypeName(result_types[0]));
-  } else {
-    printf("nil");
+  switch (result_count) {
+    case 0:
+      printf("nil");
+      break;
+    case 1:
+      printf("%s", GetTypeName(result_types[0]));
+      break;
+    default:
+      printf("(%s", GetTypeName(result_types[0]));
```

Comment:
move this statement into the loop? check for `,` with `if (i) printf(", ");` ?

Reply:
What about the parenthesis? Since I need a print statement for `(` anyway, I combined it with the first element.

Reply:
I've modified the loop to make it more standard.